<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="tableSelectLabel">
         <property name="text">
          <string>Table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="tableSelect"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="filterLabel">
         <property name="text">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="filterInput"/>
       </item>
       <item>
        <widget class="QLabel" name="sortLabel">
         <property name="text">
          <string>Sort</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sortBySelect"/>
       </item>
       <item>
        <widget class="QComboBox" name="sortOrderSelect"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableView"/>
     </item>
     <item>
      <widget class="QTextEdit" name="sqlView"/>
     </item>
     <item>
      <widget class="QLineEdit" name="errorsView">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
